src/stratis_cli/_actions/_data.py

```diff
@@ -27,6 +28,7 @@
 from dbus_python_client_gen import DPClientGenerationError
 
 from .._errors import StratisCliGenerationError
+from .._errors import StratisCliEnvironmentError
```

Comment:
alphabetize! E before G. Hopefully soon we'll be handing this detail off to a program.